DOCUMENTATION.md

```diff
@@ -776,6 +776,58 @@ If the request is not successful, the promise fails with an `err`.
 |`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: signature, api token not found"`<br>`}]`|Use the correct API key. Refer to [API keys](#api-keys) for more information|
 
 
+## Get a PDF for a letter notification
+
+### Method
+
+```javascript
+notifyClient
+  .getPdfForLetterNotification(notificationId)
+  .then(function (fileBuffer) {
+    return fileBuffer
+  }) /* the response is a file buffer, an instance of Buffer */
+  .catch((err) => console.error(err))
+```
+
+The method returns a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises) [external link]. The promise will either:
+
+- resolve with `fileBuffer` if successful
+- fail with an `err` if unsuccessful
+
+### Arguments
+
+#### notification_id (required)
+
+The ID of the notification. You can find the notification ID in the response to the [original notification method call](/python.html#get-the-status-of-one-message-response).
+
+You can also find it in your [GOV.UK Notify Dashboard](https://www.notifications.service.gov.uk).
+
+1. Sign into GOV.UK Notify and select __Dashboard__.
+1. Select __letters sent__.
+1. Select the relevant notification.
+1. Copy the notification ID from the end of the page URL, for example `https://www.notifications.service.gov.uk/services/af90d4cb-ae88-4a7c-a197-5c30c7db423b/notification/ID`.
```

Comment:
see https://github.com/alphagov/notifications-node-client/pull/95 for how this can be shortened :)